src/GitVersionTfsTask/Extension/vss-extension.json

```diff
@@ -1,9 +1,8 @@
 {
     "manifestVersion": 1,
-    "id": "gitversion",
+    "id": "gl-gitversion",
     "name": "GitVersion",
-    "publisher": "gittools",
-    "public": true,
```

Comment:
Our tooling allows you to build different version of the same extension. In order to do that we use the extension flags instead and they are configured in `configuration.json`.